### DISCUSSION TITLE
Add register 80 to turn the inverter off

### DISCUSSION THREAD

**Discussion by ap20132022** [kellerza/sunsynk#373] [Q&A]
Hi all, 
I'd like to add the register 80 (decimal) as a custom sensor, and use it like a "switch" at Home Assistant.
Can anyone help me. How can I do it? 
Thank you very much

`
from sunsynk.rwsensors import SwitchRWSensor

SENSORS += SwitchRWSensor(80, "Turn inverter Standby")
`
Do I miss anything?

**Comment by kellerza** [comment]
That definition seems ok, you can also specify a on&off value if this is different from the defaults

In my modbus docs, register 80 is something else (single phase)

**Comment by deondti** [comment]
> Hi all, I'd like to add the register 80 (decimal) as a custom sensor, and use it like a "switch" at Home Assistant. Can anyone help me. How can I do it? Thank you very much
> 
> ` from sunsynk.rwsensors import SwitchRWSensor
> 
> SENSORS += SwitchRWSensor(80, "Turn inverter Standby") ` Do I miss anything?

In lovelace , add card...
you can by switch or by entity...
or
where it's discovered in MQTT , add to dashboard, make sure it's being discovered in MQTT...

BUT.... as @kellerza  mentioned.... make sure you have the correct address

**Comment by mynkow** [comment]
I have SUN-12K-SG04LP3-EU. In my case I am not able to select Single or 3 phase inverter in HomeAssistant. It always goes to single phase no matter what I have selected. If the registry `80` is for the power ON/OFF as I am reading the [docs here](https://github.com/user-attachments/files/16090118/MODBUSRTU.V104.3.1.1111_SG01-HP3-AM2.pdf) that explains why that setting is `Single` all the time.
